spec/factories/host_genomes.rb

```diff
@@ -8,15 +8,12 @@
     # up to this point have been created by admins.
     association :user, factory: [:admin]
 
-    after :create do |host_genome, options|
+    after :create do |_host_genome, options|
       options.metadata_fields.each do |metadata_field_name|
         metadata_field = MetadataField.find_by(name: metadata_field_name)
         unless metadata_field
-          metadata_field = create(:metadata_field, name: metadata_field_name)
+          create(:metadata_field, name: metadata_field_name)
         end
-
-        # Add the metadata field to the host genome if it doesn't already exist.
```

Comment:
Should this still be removed? It seems like the metadata field wouldn't be added to the host genome unless it was required or default for host genome.